### DISCUSSION TITLE
Fixed rate speed modification mod

### DISCUSSION THREAD

**Discussion by osuShiro** [ppy/osu#17317] [Ideas] [answered]
I looked for an idea similar to this and haven't found any. I hope this isn't a duplicate.

The idea is simple: a mod that allows you to play a map at a chosen rate different from the flat 1.5 and 0.75 of DT and HT. Wind-up sort of already does this, but there must always be a 0.01 difference between initial rate and final rate. It's a negligible difference, I agree, but it would be nice to have something dedicated to modifying the beatmap's speed uniformly.

The purposes of this should be obvious: practicing higher speeds, challenging oneself, or simply playing beatmaps at comfortable BPMs.

**Comment by bdach** [comment, marked as answer]
this is already in the game. you can customise rate when selecting DT/HT.

**Reply by osuShiro** [reply]
I had not noticed the customisation tab - thank you for your quick answer. Disregard this entirely then!
